Dockerfile

```diff
@@ -1,83 +1,20 @@
 FROM ubuntu:precise
 MAINTAINER Ben Cawkwell <bencawkwell@gmail.com>
 
-# Update the system
```

Comment:
Moved this into setup.sh, it allowed some cleanup and a small reduction in image size